tests/test.sh

```diff
@@ -7,6 +7,10 @@
 
 pycodestyle ../cddl_gen/cddl_gen.py --max-line-length=120 --ignore=W191,E101,W503
 [[ $? -ne 0 ]] && exit 1
+pycodestyle ../cddl_gen/__init__.py --max-line-length=120
```

Comment:
Why 120? [PEP-8](https://www.python.org/dev/peps/pep-0008/#maximum-line-length) considers 99 to be the max when teams agree on it, otherwise 80. IMO 80 is outdated, so I'm a fan of the 99/100 personally.